### PR TITLE
856 toolbox metal test cases

### PR DESCRIPTION
**Pull request by cdelnano**
### What are the issues this pull addresses (issue numbers / links)?
[#856 Add Additional Test Cases](https://github.com/Qiskit/qiskit-metal/issues/856)

### Did you add tests to cover your changes (yes/no)?
yes

### Did you update the documentation accordingly (yes/no)?
no (added comments in test_toolbox_metal.py)

### Did you read the CONTRIBUTING document (yes/no)?
yes

### Summary
Added unit tests for the following functions/methods:
- input_error
- open_docs
- orient_me
- get_platform_info
- extract_value_unit
- aligned_pts

### Details and comments
Added unit tests to validate these functions/methods. Completed a portion of issue #856 . I would like to save this branch to continue developing on it.

### CODE REVIEW

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
This a great start to adding test cases to CI.  Thank you.

File: qiskit_metal/tests/test_toolbox_metal.py [hunk 1]
```diff
@@ -256,6 +293,11 @@ def test_toolbox_metal_parse_options(self):
                                           expected[x][i],
                                           rel_tol=1e-3)
 
+    def test_toolbox_metal_extract_value_unit(self):
+        """Test functionality of extract_value_unit in toolbox_metal.py."""
+        self.assertEqual(parsing.extract_value_unit("200", "mm"), 200.0)
```
Comment (ThomasGM4) [inline qiskit_metal/tests/test_toolbox_metal.py:298]: Think you would want the testing to be something like

`self.assertEqual(parsing.extract_value_unit("200mm", "mm"), 200.0)`

or

`self.assertEqual(parsing.extract_value_unit("200um", "mm"), 0.2)`

**Review by ThomasGM4 — CHANGES_REQUESTED** [review]

**Review by obrienpja — APPROVED** [review]
LGTM! Great job!

**Review by cdelnano — COMMENTED** [review]
Updated test_toolbox_metal_extract_value_unit per Thomas's comment.

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM, great job.

**Review by ThomasGM4 — APPROVED** [review]

**Pull request merged**